src/v3/src/transformer/selectAuthenticator/utils.ts

```diff
@@ -214,7 +190,7 @@ const formatAuthenticatorOptions = (
           ? loc('oie.enroll.authenticator.button.text', 'login')
           : loc('oie.verify.authenticator.button.text', 'login'),
         description: getAuthenticatorDescription(
-          options,
```

Comment:
Nice and simple solution -- Was there any reason we were passing the whole array of options before? It seems pretty obvious now that we should only care about the single option rather than get the whole array, so wondering if there was some sort of utility or plans for this method to do more.

Reply:
I can't think of a legitimate reason for passing the entire options array (because that just complicates things and forces you to search the array for the corresponding option, which is already known since the caller is iterating over options). 

The only thing I can think of is that with OV, you need to modify the options array (for totp and push method types) but that is handled in a completely separate isolated function that doesn't overlap with this one, so maybe they were combined at one point then refactored 🤔 (its the only logical explanation of why this was written this way lol)